core/src/main/java/com/scalar/db/storage/jdbc/RdbEngineMysql.java

```diff
@@ -201,9 +212,9 @@ public String getDataTypeForEngine(DataType scalarDbDataType) {
   public String getDataTypeForKey(DataType dataType) {
     switch (dataType) {
       case TEXT:
-        return "VARCHAR(64)";
+        return "VARCHAR(" + keyColumnSize + ")";
```

Comment:
I guess this column definition is also used for `coordinator.state.tx_id` column, which usually contains a UUID value or [a UUID value plus 25 byte prefix for the group commit feature](https://github.com/scalar-labs/scalardb/blob/master/core/src/main/java/com/scalar/db/transaction/consensuscommit/CoordinatorGroupCommitter.java#L40-L44). So, a validation that confirms the key column size is equal to or larger than 64 would be needed somewhere?

Reply:
@komamitsu Thank you! That's a really good point. We should check it maybe in `JdbcConfig`. I will add it.

Reply:
@komamitsu Revised in b04a069. PTAL!